@vmswr/vmswr.m

```diff
@@ -88,6 +85,11 @@
 		data.analogInfo.ProbeInfo = strrep(data.analogInfo.ProbeInfo,'raw','lfp');
 		data.analogTime = (0:(data.analogInfo.NumberSamples-1))' ./ data.analogInfo.SampleRate;
         
+        data.analogRmsData = nptRms(bpdata,10,5,1); % Calculate RMS every 5ms using a moving 10ms window
```

Comment:
Constants like 10, 5, and 1 should be set as optional input arguments so they can be easily changed by the user without having to edit the code.